src/core/public/chrome/ui/header/branding/opensearch_dashboards_custom_logo.tsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+import React from 'react';
+
+interface CustomLogoType {
+  logoURL: string;
+}
+
+export const CustomLogo = ({ ...branding }: CustomLogoType) => {
+  return (
+    <div>
+      <img src={branding.logoURL} alt="logo" height="30" loading="lazy" />
```

Comment:
NIT: we probably could include styling from .scss, I noticed the padding is a little off as well.

did we want to include:
```
height
line-height
min-width
padding
```

Reply:
NIT: should we include one final onError that points to the local svg for logo?

Reply:
I will talk with Kevin about finalizing this.